### ISSUE TITLE
pip install pyfinviz==0.5 results in error

### ISSUE THREAD

**Issue by zianhuang** [oscar0812/pyfinviz#1]
There's a missing comma in [setup.py](https://github.com/oscar0812/pyfinviz/blob/main/setup.py#L19)

Error:
```
ERROR: Could not find a version that satisfies the requirement numpylxml (from pyfinviz)
ERROR: No matching distribution found for numpylxml
```

Btw, this is some solid work 👍

**Comment by oscar0812** [comment]
Thanks! I will fix this asap!

**Comment by oscar0812** [comment]
Fixed in 0.6! Let me know if you have other issues!
